Marlin/src/pins/stm32f4/pins_MKS_MONSTER8_V2.h

```diff
@@ -58,6 +58,8 @@
   #define WIFI_RESET_PIN                    PD14  // MKS ESP WIFI RESET PIN
 #endif
 
-#define NEOPIXEL_PIN                        PC5
+#if DISABLED(FYSETC_MINI_12864_2_1)
+  #define NEOPIXEL_PIN                        PC5
+#endif
```

Comment:
If this board is used for a different project that includes NeoPixel lighting, would this be the first choice for `NEOPIXEL_PIN` or would they have to use another pin?

Reply:
PC5 is the motherboard labeled neopixel port

Reply:
I see. So it doesn't depend on `FYSETC_MINI_12864_2_1` specifically, but only on the general NeoPixel feature.

Reply:
All boards should be defined in the most generic terms possible. If the existence of a `NEOPIXEL_PIN` is not forcing the `NEOPIXEL_LED` option or some other feature to be enabled (which would be a throwback to the Old Ways) then a board that has a preferred `NEOPIXEL_PIN` should simply define it. Since any free GPIO pin could theoretically be used for NeoPixel, it should be overridable in the configuration. So…
```cpp
#ifndef NEOPIXEL_PIN 
  #define NEOPIXEL_PIN PC5
#endif
```